src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -175,14 +175,10 @@ public void testGetStudent() throws InvalidParametersException, EntityDoesNotExi
         s2 = createNewStudent("one.new@gmail.com");
         assertTrue(studentsDb.getUnregisteredStudentsForCourse(s2.course).get(0).isEnrollInfoSameAs(s2));
         
-        s2.googleId = null;
-        studentsDb.updateStudentWithoutSearchability(s2.course, s2.email, s2.name, s2.team, s2.section, s2.email, s2.googleId, s2.comments);
-        assertTrue(studentsDb.getUnregisteredStudentsForCourse(s2.course).get(0).isEnrollInfoSameAs(s2));
```

Comment:
These lines are not testing any functions that have not already been tested. Also, getAllStudents is deprecated and should not be used.